WIKI.org

```diff
@@ -142,7 +142,31 @@ about that - here's a screencast showing you how to do it:
  have HTTP ok status.
  #+END_EXAMPLE
 
- then the =RewriteRule= in the above example is either missing from, or ineffective in your webserver config.
+ then something is wrong with your webserver config.  You can check
+ whether a CORS preflight check is returning the right headers via:
+
+ #+BEGIN_EXAMPLE
+ curl -v -X OPTIONS https://my.server/webdav
```

Comment:
NB: For the official Apache test bench, we need to have a trailing slash, otherwise there's no 200 ok, but a 301 permanently moved.